manage.py

```diff
@@ -1,24 +1,36 @@
+import json
+import logging
+import logging.config
+import os
+
 import click
 import pidfile
 
 from ocdskingfisherarchive.archive import Archive
 from ocdskingfisherarchive.config import Config
+from ocdskingfisherarchive.database_archive import DataBaseArchive
+from ocdskingfisherarchive.database_process import DataBaseProcess
 
 config = Config()
 config.load_user_config()
 
 
 @click.group()
 def cli():
-    pass
+    logging_config_file_full_path = os.path.expanduser('~/.config/ocdskingfisher-archive/logging.json')
+    if os.path.isfile(logging_config_file_full_path):
+        with open(logging_config_file_full_path) as f:
+            logging.config.dictConfig(json.load(f))
 
 
 @cli.command()
 def archive():
     try:
         with pidfile.PIDFile():
-            archive = Archive(config)
-            archive.archive()
+            database_archive = DataBaseArchive(config)
+            database_process = DataBaseProcess(config)
+            archive_worker = Archive(config, database_archive, database_process)
```

Comment:
I'm not sure we need this degree of dependency injection, but we can leave it for now.

Reply:
I was thinking it might really help with injecting mocks during testing more bits of work, so it would be good to leave now and we can take out later if wanted.

Reply:
Created follow-up issue: https://github.com/open-contracting/kingfisher-archive/issues/21